### PR TITLE
Qr code

### PR DESCRIPTION
**Pull request by dellThejas**
This change allows for facial recognition and badge matching.

### CODE REVIEW

File: camera-recorder/src/main/java/io/pravega/example/camerarecorder/CameraRecorder.java [hunk 1]
```diff
@@ -145,6 +146,7 @@ public void run() {
 
                     // Write to Pravega.
                     CompletableFuture<Void> future = pravegaWriter.writeEvent(Integer.toString(videoFrame.camera), jsonBytes);
+//                    future = pravegaWriterCopy.writeEvent(Integer.toString(videoFrame.camera), jsonBytes);
```
Comment (claudiofahey) [inline camera-recorder/src/main/java/io/pravega/example/camerarecorder/CameraRecorder.java]: Remove commented code.

File: camera-recorder/src/main/java/io/pravega/example/camerarecorder/Test.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+import java.io.IOException;
+import java.rmi.ConnectIOException;
+
+import static org.bytedeco.opencv.global.opencv_imgcodecs.imencode;
+
+/**
+ * Reads video images from a web cam and writes them to a Pravega stream.
+ */
+public class Test implements Runnable {
```
Comment (claudiofahey) [inline camera-recorder/src/main/java/io/pravega/example/camerarecorder/Test.java]: Is this class needed? If so, give it a more descriptive name.

File: camera-recorder/src/test/java/io/pravega/example/camerarecorder/FacialRecognitionTests.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
+//package io.pravega.example.camerarecorder;
```
Comment (claudiofahey) [inline camera-recorder/src/test/java/io/pravega/example/camerarecorder/FacialRecognitionTests.java]: Remove this file.

File: common/src/main/java/io/pravega/example/tensorflow/EmbeddingsComparator.java [hunk 4]
```diff
@@ -8,6 +8,9 @@
 import java.util.Map;
 import java.util.Set;
 
+/**
+ * This class allows to build labels, compare face embeddings and compare embeddings
```
Comment (claudiofahey) [inline common/src/main/java/io/pravega/example/tensorflow/EmbeddingsComparator.java]: Fix this comment.

File: common/src/main/java/io/pravega/example/tensorflow/QRCode.java [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+        System.out.println(id);
+    }
+
+    /**
+     *
+     * @param qrCodeData data that is needed to be converted into a QR code
+     * @param filePath location where the QR code needs to be stored
+     * @param charset which type of charset needs to be used for input data
+//     * @param hintMap configuration of QR code generator
```
Comment (claudiofahey) [inline common/src/main/java/io/pravega/example/tensorflow/QRCode.java]: Fix this.

File: common/src/main/java/io/pravega/example/tensorflow/QRCode.java [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                .lastIndexOf('.') + 1), new File(filePath));
+    }
+
+    public static String readQRCode(byte[] imageBytes)
+            throws IOException, NotFoundException {
+        BinaryBitmap binaryBitmap = new BinaryBitmap(new HybridBinarizer(
+                new BufferedImageLuminanceSource(
+                        ImageIO.read(new ByteArrayInputStream(imageBytes)))));
+        Result qrCodeResult = new MultiFormatReader().decode(binaryBitmap);
+//        Result qrCodeResult = new MultiFormatReader().decode(binaryBitmap, hintMap);
```
Comment (claudiofahey) [inline common/src/main/java/io/pravega/example/tensorflow/QRCode.java]: Remove commented code.

File: common/src/main/java/io/pravega/example/tensorflow/QRScanner.java [hunk 7]
```diff
@@ -0,0 +1,109 @@
+//package io.pravega.example.tensorflow;
```
Comment (claudiofahey) [inline common/src/main/java/io/pravega/example/tensorflow/QRScanner.java]: Remove this file.

File: flinkprocessor/src/main/java/io/pravega/example/videoprocessor/TwoReaderSameStream.java [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+import org.apache.flink.streaming.api.windowing.time.Time;
+import org.apache.flink.streaming.api.windowing.windows.TimeWindow;
+import org.apache.flink.util.Collector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class TwoReaderSameStream extends AbstractJob {
```
Comment (claudiofahey) [inline flinkprocessor/src/main/java/io/pravega/example/videoprocessor/TwoReaderSameStream.java]: What is this class for?

File: jupyterhub/notebooks/install_dependencies.ipynb [hunk 9]
```diff
@@ -35,18 +35,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (claudiofahey) [inline jupyterhub/notebooks/install_dependencies.ipynb]: Revert changes to this file.

File: scripts/linux-amd64/LICENSE [hunk 10]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (claudiofahey) [inline scripts/linux-amd64/LICENSE]: Remove this file.

File: scripts/linux-amd64/README.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
+[![CircleCI](https://circleci.com/gh/helm/helm.svg?style=shield)](https://circleci.com/gh/helm/helm)
```
Comment (claudiofahey) [inline scripts/linux-amd64/README.md]: Remove this file.

File: video-player/src/main/java/io/pravega/example/videoplayer/CopyStream.java [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+import org.bytedeco.opencv.opencv_core.Mat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.ByteBuffer;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Reads video images a Pravega stream and displays them on the screen.
```
Comment (claudiofahey) [inline video-player/src/main/java/io/pravega/example/videoplayer/CopyStream.java]: Fix this comment. Remove the whole class if this is not needed.

**Review by claudiofahey — CHANGES_REQUESTED** [review]
Remove all images unless they are part of an automated test. Do not use any copyrighted images.
Remove .swo file.
Remove database.json.

File: .gitignore [hunk 13]
```diff
@@ -6,6 +6,7 @@
 *.log
 .nfs*
 build/
+scripts/linux-amd64/*
```
Comment (claudiofahey) [inline .gitignore]: Remove this line.

File: common/src/main/resources/database.json [hunk 14]
```diff
@@ -0,0 +1 @@
...0.0079337,0.005255279,0.012413879,-0.050560057,0.02680774,0.009283875,-0.066311404,-0.0051061097,0.0070918645,0.037648942,0.012220223,-0.059173595,0.039940014,-0.028772037,0.04342835,0.0064989324,-0.06832959,0.03193842,0.030271787,-0.047159314,-0.051822588,-0.035340797,0.008878209,0.010372877,0.033519026,-0.00773179,0.03598311,-0.031236239,0.056789912,-0.035287153,0.040131558,0.018791204,-0.007946787,0.048809506,-0.011039473,0.01571355,-0.004306602,-0.01688446,0.04905954,0.045990113,0.035895724,-0.06885573,-0.06437191,0.09075305,-0.016402407,-0.01232676,0.06545709,0.020123225,-0.052584656,-0.023178395,-0.06464364,0.007589575,-0.051862646,-0.024413362,-0.019487048,0.042230967,-0.0318206,-0.02036199,-0.045971736,0.050224517,-0.0414286,0.0046263444,-0.043836802,-0.003520863,-0.039544486,-0.031095546,0.017569365,-0.0035538925,-0.04279008,-0.03764251,0.024977468,-0.016619291,0.04399878,-0.02832864,0.048008617,-0.06682555,-0.005368233,-0.057917897,-0.07231655,-0.008047463,-0.019587265,0.08094477,0.008905881,-0.0074357023,0.027709207,0.025202598,0.026805889,-0.016460862,0.05909475,0.029984284,0.04772865,0.058244355,-0.004801914,0.038055968,0.06772643,-0.071798414,0.039900437,0.09695081]}]
```
Comment (claudiofahey) [inline common/src/main/resources/database.json]: Why is this file needed?

**Review by claudiofahey — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 13 above)
Comment (dellThejas) [reply .gitignore]: I added this because, my local environment kept adding linux-amd64 scripts, but I will remove it

File: common/src/main/resources/database.json (hunk 14 above)
Comment (dellThejas) [reply common/src/main/resources/database.json]: removed

**Review by claudiofahey — APPROVED** [review]
